rust/gui-client/src-tauri/src/client/network_changes/windows.rs

```diff
@@ -238,8 +242,8 @@ impl Worker {
         Ok(())
     }
 
-    pub(crate) async fn notified(&self) {
-        self.notify.notified().await;
+    pub(crate) async fn notified(&mut self) {
```

Comment:
I think using `recv` here as a name will be less surprising for someone going through this function